Week1/homework/nadin.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+let greetingIndonesian = 'Ciao';
+let greetingArabic = 'مرحبا ';
+let greetingSpanish = 'Halo';
+ console.log(greetingIndonesian, greetingArabic, greetingSpanish);
```

Comment:
Hi Nadin, you can use the `const` keyword here instead of `let`. This is because we are not re-assigning any of the greeting variables, right? 🙂

Reply:
👍